bciers/apps/dashboard/tests/routes/dashboard/page.test.tsx

```diff
@@ -0,0 +1,241 @@
+import { render, screen } from "@testing-library/react";
```

Comment:
Flagging this as I wasn't sure where to put `page.tsx` tests, so I created a `tests/routes` directory.